edx_toggles/toggles/state/internal/report.py

```diff
@@ -213,7 +213,14 @@ def _add_settings(settings_dict):
     """
     Fill the `settings_dict`: will only include values that are set to true or false.
     """
-    for setting_name, setting_value in vars(settings).items():
```

Comment:
Was the issue that `vars` just doesn't work at all with the settings object?

Reply:
It works, but only for settings that were previously requested. So if I set a new test setting, and then if I retrieve that setting in middleware, before the view that uses this report, then it starts appearing. I imagine this is due to lazy loading of settings.

Reply:
Note: I did ensure that the new test I added failed without the fix.

Reply:
Ah, very interesting. And that makes sense.